### ISSUE TITLE
Enterprise & Community v3.5.16: Log out and cant log back in 

### ISSUE THREAD

**Issue by joehoeller** [neo4j/neo4j-browser#1087]
After I create a user (with CALL function) or use default user after forced password change, in either community or enterprise v3.5x, I log out of UI with ‘:server disconnect’, I cannot log back in. 

This is very problematic and should be addressed ASAP. NOT in the mood for developer pedantry either. Let’s just admit it’s a problem and fix it.

**Comment by oskarhane** [comment]
Thanks for your report @joehoeller.

We are unable to reproduce this, could you share more about your setup / environment, or even a screen recording that shows all your steps and the result of them?

**Comment by joehoeller** [comment]
Yes I’ll make a video. Give me some time, bogged down here at work. #DeveloperLife 🙄
